### PR TITLE
add support for 1.12.x and update make install

### PR DESCRIPTION
**Pull request by pei0804**
**Describe the PR**
https://github.com/swaggo/swag/pull/326#pullrequestreview-217074161

go 1.10.x below user use `go get` .
go 1.11.x top user use `go module` .
And each version can use Makefile.

### CODE REVIEW

**Comment by pei0804** [comment]
@easonlin404 plz review.

**Comment by pei0804** [comment]
sry wait.

**Comment by pei0804** [comment]
@easonlin404 review plz.

File: Makefile [hunk 1]
```diff
@@ -48,13 +50,12 @@ install:
 	$(GOGET) -v ./...
 	$(GOGET) github.com/stretchr/testify/assert
 
+update-pkg:
+	$(GOGET) -u ./...
```
Comment (easonlin404) [inline Makefile]: It's seems like unused right?

Comment (pei0804) [reply Makefile]: Yeah. I forget delete.

Comment (pei0804) [reply Makefile]: I deleted now.

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,8 @@ BINARY_NAME:=swag
 PACKAGES:=$(shell $(GOLIST) ./...)
 GOFILES:=$(shell find . -name "*.go" -type f)
 
+export GO111MODULE := on
```
Comment (easonlin404) [inline Makefile]: It work when users `make install` or build via travis, but my only concern is that's not clear when user viewed via travis build log. https://travis-ci.org/swaggo/swag/jobs/510242966#L440-L475

Comment (easonlin404) [reply Makefile]: rest LGTM

Comment (easonlin404) [reply Makefile]: maybe  add the following in travis.yml is a good approach
```
env:
  global:
  - GO111MODULE="on"
```

Comment (easonlin404) [reply Makefile]: @pei0804 ping

Comment (pei0804) [reply Makefile]: I see.
https://github.com/swaggo/swag/pull/342#discussion_r268387322

**Pull request merged**